packages/playground/remote/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<title>Wordpress for teaching</title>
+		<link href="./src/style.css" rel="stylesheet" />
+	</head>
+
+	<body>
+		<div><x-wordpress /></div>
+		<p style="float: right">Version: <x-version /></p>
+		<script src="https://wp-mfe.pages.dev/bundle.js"></script>
+		<script src="./src/version.ts" type="module"></script>
+	</body>
+</html>
```

Comment:
This seems specific to the _Wordpress for teaching_ project and shouldn't be part of core Playground.